### DISCUSSION TITLE
I built a  social media platform with pocketbase and vanilla js

### DISCUSSION THREAD

**Discussion by MalikWhitten67** [pocketbase/pocketbase#1647] [Show and tell]
The project took me a total of 13 days - features: image & video & text posting, videos can only be 6mb in size min quality is 720p 30
Ive added keywords for emojis like grinning_emoji, and embedding youtube videos  for people who want bigger upload limits. ive allowed certain html tags in posts for those who want to use them to organize post content like making articles etc. also has several link filters and profanity filters to ensure no offending happens

Current devices supported:
pc - N/A
Mobile - any device with a width of 360, 414 , 412 and 390 work, certain phones do not atm

url: https://postr.rf.gd/

Framework I used: None - this is static html & regular js :}
![image](https://user-images.githubusercontent.com/65188863/213689669-96d1bdde-8a51-45e6-81f1-4d877bdaef7d.png)
![image](https://user-images.githubusercontent.com/65188863/213689700-016df16f-4c49-4541-be3b-c7f26a1adf64.png)

**Comment by ashermyers** [comment]
Why did you obfuscate the entire thing. The backend looks like a crime scene.

**Reply by MalikWhitten67** [reply]
In the past, I used to obfuscate my code to remove any references to the API URL because PocketHost did not provide any parameters to hide it from the user. This was particularly important because allowing users to access the API URL directly could pose a security risk to the application. However, I have since stopped using Pocketbase altogether as it proved to be too slow for my needs. I currently send at least 25,000 requests every 20 minutes during testing, and my new JSON database is much faster and more efficient. In addition, I have implemented various packages such as Cluster to act as load balancing, which has greatly improved the performance of my application.
